From 720911cee60390e0f9f39d441266cddef17e4b47 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 10 Jul 2014 01:09:11 -0400
Subject: [pci] hotplug/acpi: Drop unused acpiphp_debug declaration

Message-id: <20140710010910.22146.80050.stgit@gir.stowe>
Patchwork-id: 85643
O-Subject: [RHEL7.1 PATCH 104/105] PCI / hotplug / ACPI: Drop unused acpiphp_debug declaration
Bugzilla: 1114228
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1114228
Upstream Status: 296e5809ee567e41d55da41a882abef9d8071e5e

commit 296e5809ee567e41d55da41a882abef9d8071e5e
Author: Mika Westerberg <mika.westerberg@linux.intel.com>
Date:   Tue Nov 19 16:15:35 2013 +0200

    PCI / hotplug / ACPI: Drop unused acpiphp_debug declaration

    Commit bd950799d951 (PCI: acpiphp: Convert to dynamic debug) removed users
    of acpiphp_debug variable and the variable itself but the declaration was
    left in the header file. Drop this unused declaration.

    Signed-off-by: Mika Westerberg <mika.westerberg@linux.intel.com>
    Acked-by: Bjorn Helgaas <bhelgaas@google.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/pci/hotplug/acpiphp.h b/drivers/pci/hotplug/acpiphp.h
index 26100f5..1592dbe 100644
--- a/drivers/pci/hotplug/acpiphp.h
+++ b/drivers/pci/hotplug/acpiphp.h
@@ -176,7 +176,6 @@ u8 acpiphp_get_latch_status(struct acpiphp_slot *slot);
 u8 acpiphp_get_adapter_status(struct acpiphp_slot *slot);
 
 /* variables */
-extern bool acpiphp_debug;
 extern bool acpiphp_disabled;
 
 #endif /* _ACPIPHP_H */
-- 
1.7.1